src/shell/atuin.zsh

```diff
@@ -9,11 +9,11 @@
 # Source this in your ~/.zshrc
 autoload -U add-zsh-hook
 
-export ATUIN_SESSION=$(atuin uuid)
+export ATUIN_SESSION=$(RUST_LOG=error atuin uuid)
 export ATUIN_HISTORY="atuin history list"
```

Comment:
not sure what this is for

Reply:
It's unused. Looks like it's a leftover from an older version of the script.